backend/src/rules/play_card.clj

```diff
@@ -1,37 +1,32 @@
 (ns rules.play-card
   (:require [configs.messages :as messages]
-            [rules.alter-card :as alter-card]
             [rules.count-cards :as count-cards]))
 
-(defn move-card
+(defn move-card-to-row
```

Comment:
I think I actually prefer this option:

```clojure
(defn move-card
  "Moves a card to the specified location"
  [game-state card-id location]
  (assoc-in game-state [:cards card-id :location] location))
```

Which would be used like:

```clojure
(move-card (:card-id play) [:row (:row-id play)])
```

What was the reason for choosing this option instead? I can be convinced 🙃

Reply:
`(move-card` will only be used to move cards to row.

And
`(move-card-to-row game-state (:card-id play) (:row-id play))` is more readable than `(move-card (:card-id play) [:row (:row-id play)])`

Still, I think we can do something in between. I create the function you perefer (and test it) and
```clojure
(defn ^:private move-card-to-row
  "Moves a card to the specified row"
  [game-state card-id row-id]
  (move-card game-state card-id [:row row-id]))
```

Reply:
I'm convinced 😉